### PR TITLE
HOTFIX: 2.2 fixes

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: frontend/src/features/contacts/contact/edit/Person/UpdatePersonForm.tsx [hunk 1]
```diff
@@ -126,6 +125,17 @@ const UpdatePersonComponent: React.FC<FormikProps<IEditablePersonForm>> = ({
     }
   };
 
+  const isContactMethodInvalid = useMemo(() => {
```
Comment (devinleighsmith) [inline frontend/src/features/contacts/contact/edit/Person/UpdatePersonForm.tsx:128]: The update form is not displaying the warning when data is cleared or is etl'd with no phone/email/address.

File: frontend/src/features/leases/detail/LeasePages/payment/table/payments/PaymentsForm.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ export const PaymentsForm: React.FunctionComponent<IPaymentsFormProps> = ({
               name="securityDepositsTable"
               columns={columns}
               data={payments ?? []}
-              manualPagination={false}
+              manualPagination={true}
```
Comment (devinleighsmith) [inline frontend/src/features/leases/detail/LeasePages/payment/table/payments/PaymentsForm.tsx:80]: terms/payments should not use built in pagination.

File: frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx [hunk 3]
```diff
@@ -61,7 +63,7 @@ export const PropertyDetailsTabView: React.FunctionComponent<IPropertyDetailsTab
       <StyledSummarySection>
         <LoadingBackdrop show={loading} parentScreen={true} />
         <StyledEditWrapper className="mr-3 my-1">
-          {setEditMode !== undefined && (
+          {setEditMode !== undefined && hasClaim(Claims.PROPERTY_EDIT) && (
```
Comment (devinleighsmith) [inline frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx:66]: the edit property icon should be controlled by claims

Comment (asanchezr) [reply frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx:66]: Was this reported by a PROD user that warrants a hotfix? I was about to make the same change on a related bug fix in TEST

**Review by asanchezr — APPROVED** [review]

File: frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx (hunk 3 above)
Comment (devinleighsmith) [reply frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx:66]: yes, maybe a duplicate report in uat/test

**Pull request merged**